packages/phishing-controller/src/PhishingController.ts

```diff
@@ -714,19 +725,20 @@ export class PhishingController extends BaseController<
    * this function that prevents redundant configuration updates.
    */
   async #updateHotlist() {
+    if (this.state.hotlistLastSuccessTimestamp === 0) {
+      await this.updateStalelist();
+      return;
+    }
+
     let hotlistResponse: DataResultWrapper<Hotlist> | null;
 
     try {
       if (this.state.phishingLists.length === 0) {
         return;
       }
 
-      const lastDiffTimestamp = Math.max(
-        ...this.state.phishingLists.map(({ lastUpdated }) => lastUpdated),
-      );
-
       hotlistResponse = await this.#queryConfig<DataResultWrapper<Hotlist>>(
-        `${METAMASK_HOTLIST_DIFF_URL}/${lastDiffTimestamp}`,
+        `${METAMASK_HOTLIST_DIFF_URL}/${this.state.hotlistLastSuccessTimestamp}`,
```

Comment:
What if the `hotlistLastSuccessTimestamp` is still set to `0`? Shouldn't we use the `state.phishingLists[].lastUpdated`?